### ISSUE TITLE
Asset Issues

### ISSUE THREAD

**Issue by A-c0rN** [SuperV1234/SSVOpenHexagon#364]
Hey There! I tried to compile the game from source, and everything seemed fine, but the assets were missing:

```
[Steam]                               Initializing Steam API
[Steam]                               Failed to initialize Steam API
[Discord]                             Successfully initialized core
[Discord]                             Successfully registered command
[Discord]                             Successfully registered Steam app
[::createFolderIfNonExistant]         'Profiles/' folder does not exist, creating
[::createFolderIfNonExistant]         'Replays/' folder does not exist, creating
[::loadConfig]                        loading config
[hg::Config::root()]                  User-defined `config.json` file found
[::reapplyResolution]                 reapplying resolution
[::loadAssets]                        Workshop cache file does not exist. No workshop packs to load
[::loadAssets]                        loading 'ohvrworkshopexample_vittorio_romeo_example_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_tutorial_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_hypercube_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_orthoplex_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_experimental_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_cube_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_base_1' assets
[::loadAssets]                        loading local profiles
[HGAssets::HGAssets]                  Loaded all assets in 118ms
[hg::HexagonClient::HexagonClient]    Initializing client...
 - '_serverIp': '139.162.199.162'
 - '_serverPort': '50505'
 - 'sKeyPublic': ''
 - 'sKeySecret': ''
[hg::HexagonClient::initializeTicketSteamID] Waiting for Steam ID validation...
[hg::HexagonClient::fail]             [ERROR] Never got Steam ID validation response
[hg::HexagonClient::HexagonClient]    [ERROR] Failure initializing client, no ticket Steam ID
[Steam]                               Too many failed callbacks. Stopping Steam callbacks.
[::loadAssets]                        loading local profiles
[hg::AudioImpl::playMusic]            No path for music id 'ohvrvanilla_vittorio_romeo_tutorial_1_drFinkelfracken'
[MusicData::playSeconds]              Failed playing music 'ohvrvanilla_vittorio_romeo_tutorial_1_drFinkelfracken'
[hg::HexagonGame::shouldSaveScore()]  Not saving score - in tutorial mode
[::mainClient]                        Finished
[MenuGame::~MenuGame]                 Cleaning up menu resources...
[HexagonGame::~HexagonGame]           Cleaning up game resources...
[hg::HexagonClient::~HexagonClient]   Uninitializing client...
[hg::HexagonClient::disconnect]       Disconnecting client...
[hg::HexagonClient::disconnect]       Client disconnected
[hg::HexagonClient::~HexagonClient]   Client uninitialized
[::main]                              Saving all local profiles...
[::main]                              Done saving all local profiles
[HGAssets::~HGAssets]                 Cleaning up assets...
[::main]                              Shutting down ImGui...
[::main]                              Done shutting down ImGui...
[::main]                              Saving config...
[::saveConfig]                        saving config
[::main]                              Done saving config
[::main]                              Saving log to 'log.txt'...
```

So, after reading the info and such, I ran the `./wget-assets.sh ./_RELEASE/` file, and got Assets!
However, The game followed up with this:

```
[Steam]                               Initializing Steam API
[S_API] SteamAPI_Init(): Loaded '/home/ana/.steam/debian-installation/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; connect to global user failed.[Steam]                               Failed to initialize Steam API
[Discord]                             Successfully initialized core
[Discord]                             Successfully registered command
[Discord]                             Successfully registered Steam app
[::loadConfig]                        loading config
[hg::Config::root()]                  User-defined `config.json` file found
[::reapplyResolution]                 reapplying resolution
[::loadAssets]                        Workshop cache file does not exist. No workshop packs to load
[::loadAssets]                        loading 'ohvrworkshopexample_vittorio_romeo_example_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_tutorial_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_hypercube_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_orthoplex_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_experimental_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_cube_1' assets
[::loadAssets]                        loading 'ohvrvanilla_vittorio_romeo_base_1' assets
[::loadAssets]                        loading local profiles
[HGAssets::HGAssets]                  Loaded all assets in 61lms
[hg::HexagonClient::HexagonClient]    Initializing client...
 - '_serverIp': '139.162.199.162'
 - '_serverPort': '50505'
 - 'sKeyPublic': ''
 - 'sKeySecret': ''
[hg::HexagonClient::initializeTicketSteamID] Waiting for Steam ID validation...
[hg::HexagonClient::fail]             [ERROR] Never got Steam ID validation response
[hg::HexagonClient::HexagonClient]    [ERROR] Failure initializing client, no ticket Steam ID
Segmentation fault (core dumped)
```

That didn't look right, so I re-installed, and had the same effect. I then tried to pull from the SSVOpenHexagonAssets repo, and got the same result. It just seems like assets REALLY don't like the game on this PC.

PC Specs are as follows:
```
ana@Ana-Linux 
OS: Kubuntu 21.04 x86_64 
Host: Inspiron 7720 
Kernel: 5.11.0-40-generic 
Shell: bash 5.1.4 
CPU: Intel i7-3630QM (8) @ 3.400GHz 
GPU: NVIDIA GeForce GT 650M 
Memory: 5886MiB / 11899MiB 
```

If anyone has any idea, please shoot me a message. This is a great game I have been playing since 1.92 on Puppy Linux (Oh god no) and would love to play again, preferably WITH the epic soundtrack.

**Comment by SuperV1234** [comment]
Sorry for the trouble, `./wget-assets.sh` is currently very outdated. The best option is to copy-paste the assets from the Steam version in your build folder.

**Comment by SuperV1234** [comment]
I've removed the obsolete `wget-assets.sh` file and updated the wiki to mention copying the Steam assets for now. I will look into a better solution for the future.

**Comment by nathanfranke** [comment]
tl;dr the game is free but the assets aren't
